lib/matplotlib/path.py

```diff
@@ -19,6 +19,59 @@
 from .cbook import _to_unmasked_float_array, simple_linear_interpolation
 
 
+class PathCollection:
```

Comment:
Perhaps another name would avoid confusion with `collections.PathCollection`?

Reply:
Sure, name isn't important for me. I didn't know this one is taken already. Any suggestions?

Reply:
Nothing comes to my mind for now, but we can change this later (before this is merged).